types/src/crypto/asymmetric_key/gens.rs

```diff
@@ -10,6 +10,7 @@ use proptest::{
 
 use crate::{crypto::SecretKey, PublicKey};
 
+/*
```

Comment:
I guess this code block should be removed?

Reply:
Code block removed